### PR TITLE
Home API deprecation in favor of folders personal

### LINKED ISSUE #19: Update api GET /home to be replaced wtih /folders/personal.
GET /home api is being deprecated and we need to update the api and sdk to GET /folders/personal. 

See thread in sdk_maintainers for more info.

### PR DESCRIPTION
**Pull request by timothodge**
### Description
- Removal of Home api call. In favor of folders/personal that will give back a corresponding object. 

- Also, cleaning up any references to home API to help users. 

### Testing screenshots

Screenshot 2024-04-29 at 1 28 10 PM
Screenshot 2024-04-29 at 1 28 18 PM

### CODE REVIEW

File: smartsheet-csharp-sdk/main/Smartsheet/Api/Models/PersonalFolder.cs [hunk 1]
```diff
@@ -21,9 +21,9 @@
 namespace Smartsheet.Api.Models
 {
     /// <summary>
-    /// Represents the Home object. </summary>
-    /// <seealso href="http://help.Smartsheet.com/customer/portal/articles/522237-the-home-tab">Home Tab Help</seealso>
```
Comment (timothodge) [inline smartsheet-csharp-sdk/main/Smartsheet/Api/Models/PersonalFolder.cs:25]: Shouldn't reference home tabs if we don't allow home api.

File: smartsheet-csharp-sdk/main/Smartsheet/Api/Models/PersonalFolder.cs [hunk 2]
```diff
@@ -104,15 +99,5 @@ public IList<Template> Templates
             get { return templates; }
             set { templates = value; }
         }
-
```
Comment (timothodge) [inline smartsheet-csharp-sdk/main/Smartsheet/Api/Models/PersonalFolder.cs:107]: Public API does not say folders/personal gives back workspaces so removing this. https://smartsheet.redoc.ly/tag/homeDescription

File: smartsheet-csharp-sdk/main/Smartsheet/Api/Internal/HomeResourcesImpl.cs [hunk 3]
```diff
@@ -103,7 +103,7 @@ public virtual Home GetHome(IEnumerable<HomeInclusion> includes, IEnumerable<Hom
             {
                 parameters.Add("exclude", QueryUtil.GenerateCommaSeparatedList(excludes));
             }
-            return this.GetResource<Home>("home" + QueryUtil.GenerateUrl(null, parameters), typeof(Home));
+            return this.GetResource<PersonalFolder>("folders/personal" + QueryUtil.GenerateUrl(null, parameters), typeof(PersonalFolder));
```
Comment (timothodge) [inline smartsheet-csharp-sdk/main/Smartsheet/Api/Internal/HomeResourcesImpl.cs:106]: This is the key part where we stop calling the Home public API endpoint.

**Comment by SmarMykel** [comment]
Nice bit of changes. LGTM!

**Review by SmarMykel — APPROVED** [review]

**Review by SmarMykel — APPROVED** [review]

**Review by SmarMykel — APPROVED** [review]

**Review by priyankalakhe — APPROVED** [review]

**Review by mattmeyerink — APPROVED** [review]

**Pull request merged**